onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -2471,6 +2468,68 @@ TEST_F(GraphTransformationTests, MatMulAddFusion_MissingShape) {
   ASSERT_EQ(op_to_count["Gemm"], 0);
 }
 
+TEST_F(GraphTransformationTests, MatMulAddFusion_NeedReshape_1D) {
```

Comment:
The model in "matmul_add_fusion/3Input/neg_model.onnx" has a 1D first input to the MatMul. Is this covering something different to MatMulAddFusion_negitive_case?

Reply:
You are right. Remove this one.